docs/dev/opensearch-relevancy-search.md

```diff
@@ -0,0 +1,294 @@
+* **auto_generate_synonyms_phrase_query:** (boolean) If true, match phrase queries are automatically created for multi-term synonyms. Defaults to true.
+* **fuzziness**: (string) Maximum edit distance allowed for matching.
+* **max_expansions:** (integer) Maximum number of terms to which the last provided term of the `query` value will expand. Defaults to 50.
+* **prefix_length**: (integer) Number of beginning characters left unchanged for fuzzy matching. Defaults to 0.
+* **fuzzy_transpositions**: (boolean) If true, edits for fuzzy matching include transpositions of two adjacent characters (ab → ba). Defaults to true.
+* **fuzzy_rewrite**: (string) Method used to rewrite the query.
+* **lenient**: (boolean) If true, format-based errors, such as providing a text query value for a numeric field, are ignored. Defaults to false.
+* **operator**: (string) Boolean logic used to interpret text in the query value. Available operators: (default) OR | AND.
+* **minimum_should_match**: (string) Minimum number of clauses that must match for a document to be returned.
+* **zero_terms_query**: (string)  Indicates whether no documents are returned if the analyzer removes all tokens, such as when using a stop filter. Available values: (default) none | all.
+* **boost**: (float) Boost value toward the relevance score. Defaults to 1.0.
+* **type_breaker**: (float) Floating point number between 0 and 1.0 used to increase the relevance scores of documents matching multiple query clauses. Defaults to 0.0.
+* **cutoff_frequency**: (float) Specifies an absolute or relative document frequency where high frequency terms are moved into an optional subquery and are only scored if one of the low frequency (below the cutoff) terms in the case of an or operator or all of the low frequency terms in the case of an and operator match. The `cutoff_frequency` value can either be relative to the total number of documents if in the range [0..1) or absolute if greater or equal to 1.0.
+* **slop:** (integer) **** maximum number of positions allowed between matching tokens. Defaults to 0.
+
+
+The following links are redirecting to the issues pages of the search functions details including syntax, functionality and available paramters:
+
+**1. match function**
+https://github.com/opensearch-project/sql/issues/184
+
+**2. match_phrase function**
+https://github.com/opensearch-project/sql/issues/185
+
+**3. match_phrase_prefix function**
+https://github.com/opensearch-project/sql/issues/186
+
+**4. match_bool_prefix function**
+https://github.com/opensearch-project/sql/issues/187
+
+**5. multi_match function**
+https://github.com/opensearch-project/sql/issues/188
+
+**6. combined_fields function**
+https://github.com/opensearch-project/sql/issues/189
+
+**7. common function:**
+https://github.com/opensearch-project/sql/issues/190
```

Comment:
As I mentioned above, those functions are not (and won't be) implemented in SQL.